Poker.BE/Poker.BE.Domain.Tests/Game/TestableCard.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Poker.BE.Domain.Game;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Poker.BE.Domain.Tests.Game
+{
+	class TestableCard : Card
```

Comment:
why do we need `TestableCard` class? if you cant test the `Card` class (the original one), so we have a problem that needs to be fixed (maybe the use of visibility modifiers of your Properties / Fields is wrong?